spec/services/hyrax/collections/collection_member_query_service_spec.rb

```diff
@@ -0,0 +1,49 @@
+RSpec.describe Hyrax::Collections::CollectionMemberQueryService, clean_repo: true, with_nested_reindexing: true do
+  let(:blacklight_config) { CatalogController.blacklight_config }
+  let(:repository) { Blacklight::Solr::Repository.new(blacklight_config) }
+  let(:current_ability) { instance_double(Ability, admin?: true) }
+  let!(:nestable_collection) { create(:public_collection, collection_type_settings: [:nestable]) }
+  let(:scope) { double('Scope', can?: true, current_ability: current_ability, repository: repository, blacklight_config: blacklight_config, collection: nestable_collection) }
```

Comment:
Why does scope need to respond to `can?`. Shouldn't it be using `current_ability.can?`?